cmd/node/node.go

```diff
@@ -243,7 +244,7 @@ func (app *SpacemeshApp) introduction() {
 func (app *SpacemeshApp) Initialize(cmd *cobra.Command, args []string) (err error) {
 	// exit gracefully - e.g. with app Cleanup on sig abort (ctrl-c)
 	signalChan := make(chan os.Signal, 1)
-	signal.Notify(signalChan, os.Interrupt)
+	signal.Notify(signalChan, os.Interrupt, syscall.SIGTERM)
```

Comment:
Let's try to be consistent, either use `os.Interrupt, os.Kill` or `syscall.SIGINT, syscall.SIGTERM`.

I personally prefer the second option as I find it less abstract and that it makes it clearer what we're doing.

Reply:
@noamnelke 

Consistent values look better, however, looking from the other side, `os.Kill` differs from `syscall.SIGTERM` and `syscall.SIGTERM` [is not generated in Windows](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/signal?view=msvc-160&viewFallbackFrom=vs-2017), so `os.Interrupt, syscall.SIGTERM` kind of indicates that `os.Interrupt` is intended to be caught on all systems and `syscall.SIGTERM` is needed only for systems that use it.

Also, in theory, `os.Interrupt` may be a dummy signal on an abstract system that doesn't support `SIGINT`.

I think it's a matter of taste and we can use any option.